src/polyswarm/base.py

```diff
@@ -41,8 +40,6 @@ def polyswarm(ctx, api_key, api_uri, output_file, output_format, color, verbose,
     This is a PolySwarm CLI client, which allows you to interact directly
     with the PolySwarm network to scan files, search hashes, and more.
     """
-
-    # TODO shouldn't need to do this here
```

Comment:
According with https://click.palletsprojects.com/en/7.x/complex/, it seems like to you have to initialize `ctx.obj`, so removing this comment.